bentoml/_internal/io_descriptors/multipart.py

```diff
@@ -83,7 +100,41 @@ async def from_http_request(self, request: Request) -> MultipartIO:
             res[k] = v
         return res
 
-    async def to_http_response(self, obj: MultipartIO) -> Response:
+    @t.overload
+    async def to_http_response(self, obj: t.Dict[str, str]) -> Response:  # noqa: F811
```

Comment:
Seems like pyflakes is supposed to detect overload now---maybe it has to not be qualified?

And what's the point of this, anyway? Why not just have a union type? If there isn't separate documentation I don't think it's very valuable.

Reply:
Union type doesn't seem to work with pyright

Reply:
I haven't seen an issue with that? What was wrong with it?